flipperdevices/flipperzero-firmware
# [FL-1912, FL-1939] Sub-GHz frequency analyzer and add new protocol

# What's new

- ToolBox: add manchester-decoder and manchester-encoder
- SubGhz: add new FM config cc1101
- Subghz: add protocol Kia
- SubGhz: fix receiving the last packet Nero Radio 
- SubGhz: app protocol CAME Twin (TW2EE/TW4EE)
- SubGhz: add protocol CAME Atomo (AT03EV/ AT04EV)
- [FL-1912] Sub-GHz: add frequency analyzer 
- [FL-1939] GubGhz: Frequency analyzer redesign

# Verification 

- Build, flash
- Сheck the reception of new protocols (remotes of the corresponding manufacturers are required)
- Start the frequency analyzer, press the transmission button on the relay (the key fob should be no further than 10 cm from the filpper) check how it determines the transmission frequency

# Checklist (do not modify)



[FL-1912]: https://flipperzero.atlassian.net/browse/FL-1912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FL-1939]: https://flipperzero.atlassian.net/browse/FL-1939?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
